### ISSUE TITLE
crane: image-tags of `gcr.io/go-containerregistry/crane/debug` are missing

### ISSUE THREAD

**Issue by PatrickSpies** [google/go-containerregistry#2036]
### Describe the bug

Since tonight, most image-tags of `gcr.io/go-containerregistry/crane/debug` seem to be missing.

### To Reproduce
`crane ls gcr.io/go-containerregistry/crane/debug`

> c195f151efe3369874c72662cd69ad43ee485128
> latest
> sha256-121dab6291699b59893bf2089eb3bccee37557ff4dd816f225cb3580f06d7f71.sbom
> sha256-662e32e521ffca917d40895f1bf022f0f86b31bed38b924dc256e734c9670a2c.sbom
> sha256-7034db15bc1023f9366e8e76de808cfc28ae5f8ddf68bf574a34bb610cc185d2.sbom
> sha256-94f61956845714bea3b788445454ae4827f49a90dcd9dac28255c4cccb6220ad.sbom
> sha256-d5848fb72528cb4b5899e333ede0e5d2687a661287dbf5b0cb93d16518fb3100.sbom
> sha256-d77f6f483c7d5c872b6733f6851f16ecff6907f963fda73889aaf8148800bf57.sbom

### Expected behavior

E.g. `gcr.io/go-containerregistry/crane/debug:v0.15.2` is missing, which did exist yesterday..

### Additional context

-

**Comment by davmacario** [comment]
Apparently even non-`debug` images are missing their tags: https://console.cloud.google.com/artifacts/docker/go-containerregistry/us/gcr.io/crane?inv=1&invt=Abj6_Q

**Comment by Jasper-Ben** [comment]
Yeah, no idea what is going on with the container images here. Apparently, most of the staff who initially maintained this repo at google have left the company. Since then, no one seems to have access to the CI server (and probably the gcr registry for that matter). The CI server builds already broke a while back, which is why the registry previously didn't contain up-to-date images (https://github.com/google/go-containerregistry/issues/1877). I have submitted a PR to migrate to github actions a while back but that hasn't been merged yet: https://github.com/google/go-containerregistry/pull/2000

**Comment by jhlmco** [comment]
@Jasper-Ben are there not any maintainers that can accept PR's?  Having the same problem ya'll are having and would love to help get it fixed.

**Comment by yavor-martian** [comment]
This can be now closed, right?
